includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -687,37 +803,81 @@ public function wc_setup_shipping_save() {
 	}
 
 	/**
-	 * Simple array of gateways to show in wizard.
+	 * https://stripe.com/global
+	 */
+	protected function is_stripe_supported_country( $country_code ) {
+		$stripe_supported_countries = array(
```

Comment:
Holding country names here seems pointless. They are not used.

Reply:
Fixed in a97ecb712b5cbd1cb542984a69b1964006631c85.